source/guide/3.2/guide-security.md

```diff
@@ -11,33 +11,230 @@ pageord: 500
 
 # Overview
 
-This guide will quickly explain how to bootstrap a secured manager and use if from the cli and web UI.
+This guide will explain how to bootstrap a secured manager and use it from the CLI and web UI. <br/>
+Securing the manager focuses on the REST service, which is the access point of all clients to the management server.
+
+Cloudify's security framework uses the [Flask-secuREST](https://github.com/cloudify-cosmo/flask-securest/tree/0.6)
+package, which integrates with [Flask-RESTful](https://flask-restful.readthedocs.org/en/0.3.2/) to secure REST services.
+
+When security is enabled, all requests to the REST service are authenticated before they can reach their endpoint.
+For example, when a user of the Web-UI attempts to see the all blueprints, a request is sent to the REST service
+"blueprints" endpoint, but it will only reach it if the user is logged in with valid credentials. <br/>
+Similarly, the CLI command "cfy deployments create" will only be successful if valid
+username and password are sent with the REST call to the endpoint "deployments".
```

Comment:
ah ok
i didnt see anywhere this is mentioned (CLI being able to use only user-password), so without knowing it, others could find this line a bit confusing perhaps? then again probably not.
might want to mention it anyway
up to you :D